docs/core/tutorials/using-on-mac-vs-full-solution.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Building a complete .NET Core solution on macOS using Visual Studio for Mac
```

Comment:
```suggestion
title: Build a complete .NET Core solution using Visual Studio for Mac
```

Reply:
"Create" might be a better verb.

Reply:
You have a point. I looked at the other articles, and they use "build". I'll merge this as is, and we can update all titles later if we want.